### DISCUSSION TITLE
lerna version ignore packages

### DISCUSSION THREAD

**Discussion by Marvin1003** [lerna/lerna#3341] [Q&A]
How to ignore certain packages from being versioned? I have the case where I only want to prerelease a package but exclude it from the separate graduate step. I though `--ignore-changes` would do the job, but it is still graduating the package to the stable version.

**Comment by ghiscoding** [comment]
try with [`--scope`](https://github.com/lerna/lerna/tree/main/core/filter-options#--scope-glob)
